src/shared/components/data-table-field.tsx

```diff
@@ -366,10 +386,33 @@ export const DataTableField: React.FC<FieldProps> = props => {
 
   return (
     <div className={css.dataTable}>
-      {sensor && <SensorComponent sensor={sensor}/>}
-      <div className={css.title}>{title}</div>
+      <div className={css.menu}>
+        <button className={css.saveButton} onClick={handleSaveButton}>Save</button>
```

Comment:
I removed this button and put it in the header across all experiments in the mobile app.